packages/bus-rdf-join/lib/ActorRdfJoin.ts

```diff
@@ -279,7 +279,7 @@ export abstract class ActorRdfJoin
       parentPhysicalQueryPlanNode = action.context.get(KeysInitQuery.physicalQueryPlanNode);
       action.context = action.context.set(KeysInitQuery.physicalQueryPlanNode, action);
     }
-
```

Comment:
Unneeded changes?

Reply:
Yep, I noticed my whole code base is a bit messed up. I'll open a new pull request which is cleaner hopefully